data_tests/test_data.py

```diff
@@ -71,6 +72,44 @@ def test_duplicate_entries(self):
                 self._assertTrue(data_test.passed, f"{self} [{short_path}]", short_message, full_message)
 
 
+class MissingValuesTest(TestCase):
+    def test_missing_values(self):
+        for csv_file in get_csv_files(TestCase.root_path):
+            short_path = os.path.relpath(csv_file, start=TestCase.root_path)
+
+            with self.subTest(msg=f"{short_path}"):
+                tests = []
+                with open(csv_file, "r") as csv_data:
+                    reader = csv.reader(csv_data)
+                    headers = next(reader)
+
+                    tests.append(MissingValue("county", headers))
+                    tests.append(MissingValue("precinct", headers))
+                    tests.append(MissingValue("office", headers))
```

Comment:
@dwillis, I noticed that many of the missing `office` failures are for things like "Ballots Cast" and "Registered Voters".  In some files, these appear as the `office`, and in others they appear as the `candidate` (and hence `office` is missing).

Do we have a convention for which column these types of entries should be assigned to?

Reply:
@warwickmm Ballots Cast, Registered Voters and similar values should be in `office`, not candidate. It's not a perfect fit, but it fits better there than in candidate.

Reply:
Thanks.  These can be easy issues for new volunteers to take on.